projects/irssi/build.sh

```diff
@@ -18,8 +18,10 @@
 # configure script needs leak checking disabled to not fail
 export ASAN_OPTIONS=detect_leaks=0
 ./autogen.sh
+# TODO: Stop using LIB_FUZZING_ENGINE_OLD and get this working with
+# LIB_FUZZING_ENGINE.
 ./configure --with-perl=no --disable-shared --without-textui --with-fuzzer \
-	--with-fuzzer-lib=$LIB_FUZZING_ENGINE \
+	--with-fuzzer-lib=$LIB_FUZZING_ENGINE_OLD \
```

Comment:
Shouldn't LIB_FUZZING_ENGINE_OLD be LIB_FUZZING_ENGINE_DEPRECATED ?

Reply:
DEPRECATED may be better than OLD but I intended to do old. Something got messed up when I was playing with different branches.

Reply:
OK changed it.